### PR TITLE
PE-451: Implement Context.getRequiredParameterValue()

### PR DESCRIPTION
**Pull request by matibat**

### CODE REVIEW

File: src/CLICommand/common_context.ts [hunk 1]
```diff
@@ -11,26 +11,29 @@ import {
 import { DriveID, DriveKey } from '../types';
 
 /**
- * @type {CommonContext}
+ * @type {Context}
  * A class representing the context of the parameters
  */
-export class CommonContext {
+export class Context {
```
Comment (arielmelendez) [inline src/CLICommand/common_context.ts:17]: This name is far too general. If I just asked any of our developers randomly, "hey, what do you think a context object does?", I doubt their answer would have anything to do with parameter parsing.

File: src/CLICommand/common_context.ts [hunk 2]
```diff
@@ -78,4 +81,19 @@ export class CommonContext {
...
+
+	/**
+	 * @param {ParameterName} parameterName
+	 * @returns {string | undefined}
+	 * Returns the string value for the specific parameter; returns undefined if not set
+	 */
+	public getRequiredParameterValue(parameterName: ParameterName): string | undefined {
+		const value = this.options[parameterName];
+		if (!value) {
+			throw new Error(`Parameter ${parameterName} wasn't provided!`);
```
Comment (arielmelendez) [inline src/CLICommand/common_context.ts:93]: Might want to say `Required parameter...`

File: src/CLICommand/common_context.ts (hunk 1 above)
Comment (arielmelendez) [reply src/CLICommand/common_context.ts:17]: Maybe it should be called a "ParameterHelper"?

Comment (matibat) [reply src/CLICommand/common_context.ts]: Agreed: 65d9cb76fba080a2cd9a3783f1de88ab9afc08af

File: src/CLICommand/common_context.ts (hunk 2 above)
Comment (matibat) [reply src/CLICommand/common_context.ts]: Updates the logged message: 8353bff0976349bd9813d46e92b3de6fbfce3684

**Comment by matibat** [comment]
This feature is moved to PR #55

**Pull request closed without merging**